### PR TITLE
Dev.fic msa styling

### PR DESCRIPTION
**Pull request by beth-panx**
#### What does this PR do? Please provide some context
Resolve UI changes planned for sprint 49
Task [109325 Registration Confirmation Page - Need to update yellow box message](http://vstfpg03:8080/tfs/LEXENGR1/Common/_workitems/edit/109325)
Task [109329 Account Update Required Page - Update text with latest](http://vstfpg03:8080/tfs/LEXENGR1/Common/_workitems/edit/109329)
Task [109312 Sign in page - form field missing border](http://vstfpg03:8080/tfs/LEXENGR1/Common/_workitems/edit/109312)

#### Where should the reviewer start?
Anywhere...

#### How can this be manually tested? (brief repro steps and corpnet-URL with change)
Make sure your browser remembers some account.
localhost:8000/login for task 109312
localhost:8000/register for task 109325
localhost:8000/account/link for task 109329

#### What are the relevant TFS items? (list id numbers)
[109325](http://vstfpg03:8080/tfs/LEXENGR1/Common/_workitems/edit/109325)
[109329](http://vstfpg03:8080/tfs/LEXENGR1/Common/_workitems/edit/109329)
[109312](http://vstfpg03:8080/tfs/LEXENGR1/Common/_workitems/edit/109312)

#### Definition of done:
- [x] Title of the pull request is clear and informative
- [x] Add pull request hyperlink to relevant TFS items
- [ ] For large or complex change: schedule an in-person review session
- [ ] This change has appropriate test coverage
- [ ] Get at least two approvals

#### Reminders DURING merge
1. If you're merging from a short-term (feature) branch into a long-term branch (like dev, release, or master) then "Squash and merge" to keep our history clean.
1. If merging from two longterm branches (like cherry picks from upstream, dev to release, etc) then "Create merge commit" to preserve individual commits.

[//]: # ( fyi: This content was heavily inspired by )
[//]: # ( 1 Our team's policies and processes )
[//]: # ( 2 https://github.com/sprintly/sprint.ly-culture/blob/master/pr-template.md )
[//]: # ( 3 The book "The Checklist Manifesto: How to Get Things Right" by Atul Gawande )
[//]: # ( 4 https://github.com/Azure/azure-event-hubs/blob/master/.github/PULL_REQUEST_TEMPLATE.md )

### CODE REVIEW

**Comment by sdolenc** [comment]
notes to reviewers:
- i verified that these files are not overriden by our theme so edx-platform is an appropriate place for these changes
- i expect tests will fail, but tests were failing in _msa branch prior to this change so we shouldn't block on that. i have a task to resolve failing tests that i'm working on now

File: lms/templates/student_account/link_account.underscore [hunk 1]
```diff
@@ -5,16 +5,18 @@
                 <h1><%- gettext("Account update required") %></h1>
                 <div class="account-settings-header">
                     <p class="link-account-header-subtitle"><%- userName %></p>
-                    <p class="link-account-header-subtitle"><%- gettext("In order to provide you with better security and easier profile management, we are changing to Microsoft accounts for authentication") %></span>
-                    <p class="link-account-header-subtitle"><%- gettext("Sign in with a new or existing Microsoft account, and your Microsoft Learning profile will be updated to use that account") %></span>
-                    <p class="link-account-header-subtitle"><%- gettext("This will update your login and your name and email information, but your public username will remain the same.") %></span>
+                    <p class="link-account-header-subtitle"><%- gettext("To comply with the General Data Protection Regulation (GDPR) and to better protect your privacy, you are now required to sign in with a Microsoft account. (If you don’t have one, you’ll be able to create one during the sign-in process.)") %></span>
```
Comment (sdolenc) [inline lms/templates/student_account/link_account.underscore]: nit: I would personally remove the parenthesis. the statement isn't an optional clarification, but something of equal importance IMO

File: lms/templates/student_account/link_account.underscore [hunk 2]
```diff
@@ -5,16 +5,18 @@
...
+                    <p class="link-account-header-subtitle"><%- gettext("Your name and email will be updated with the information in your Microsoft account, but your public username will remain the same.") %></span>
                 </div>
                 <div class="account-settings-section-body link-account-social-field-section">
                     <div class="ui-loading-error is-hidden">
                         <span class="fa fa-exclamation-triangle message-error" aria-hidden="true"></span>
                         <span class="copy"><%- gettext("An error occurred. Please reload the page.") %></span>
                     </div>
                 </div>
+                <div class="link-account-below-button-link" >
+                    <a target="_blank" href="https://www.microsoft.com/en-us/TrustCenter/Privacy/gdpr/default.aspx">Read more about GDPR</a>
```
Comment (btelnes) [inline lms/templates/student_account/link_account.underscore]: Nit: Might consider using a FWLink rather than a hardcoded URL.

Comment (satyarapelly) [reply lms/templates/student_account/link_account.underscore]: Need to pass from the context?

Comment (beth-panx) [reply lms/templates/student_account/link_account.underscore]: I think if this is temp solution.. we should keep it as is.

**Pull request merged**